nixos/lib/test-driver/test-driver.py

```diff
@@ -198,7 +198,7 @@ def retry(fn: Callable, timeout: int = 900) -> None:
 
 
 def _perform_ocr_on_screenshot(
-    screenshot_path: str, model_ids: Iterable[int]
+    screenshot_path: Union[str, Path], model_ids: Iterable[int]
```

Comment:
can this simply be a `Path` without the string option?

Reply:
I didn't want to break the original interface in case someone depended on that, but... Maybe?